src/ceph-volume/plugin/zfs/ceph_volume_zfs/main.py

```diff
@@ -0,0 +1,140 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import print_function
+import argparse
+import os
+import sys
+import logging
+
+import main
+from ceph_volume import log, conf, configuration
+from ceph_volume import exceptions
+from ceph_volume import terminal
+
+if __name__ == '__main__':
+    main.ZFSVOL()
+
+
+class ZFSVOL(object):
+
+    help_menu = 'Deploy OSDs with ZFS'
+    _help = """
+Use ZFS as the underlying technology for OSDs
+
+--verbose   Increase the verbosity level
+    """
+    name = 'zfs'
+
+    def __init__(self, argv=None, parse=True):
+        self.mapper = {
+        }
+        if argv is None:
+            self.argv = sys.argv
+        else:
+            self.argv = argv
+        if parse:
+            self.main(self.argv)
+        print(self.argv)
```

Comment:
@wjwithagen is this just for debugging purpose?

Reply:
@tchaikov 
I guess so.... 
There is no equivalent in the lvm code.
I'll remove that